content/articles/2019-06-13-first-gsoc-weeks.md

```diff
@@ -0,0 +1,130 @@
+Title: First GSOC weeks and poliastro's new features 
```

Comment:
I propose you use a more specific title, since @Sedictious also spent her "first weeks" :) (remember to change both the title, the slug, and the file name)

What about "Lambert maneuvers and trails in plots"?

Reply:
Sure! It is nicer and adjusts more to the topic :smile: